include/envoy/stream_info/stream_info.h

```diff
@@ -76,8 +76,44 @@ enum ResponseFlag {
 struct ResponseCodeDetailValues {
   // Response code was set by the upstream.
   const std::string ViaUpstream = "via_upstream";
-
-  // TODO(#6542): add values for sendLocalReply use-cases
+  // configured limits.
+  const std::string REQUEST_PAYLOAD_TOO_LARGE = "request_payload_too_large";
```

Comment:
nit: I think the preference is to use PascalCase for these constants.

Reply:
PascalCase for enums for sure.  For consts "In the Envoy codebase we use ConstantVar or CONSTANT_VAR."

That said I thought ALL_CAPS was strictly prefered and I prefer PascalCase so I'll switch nonetheless :-P

Reply:
I had argued for a slight preference for constants to avoid collisions with macros in files we don't control. That's codified mildly in https://github.com/envoyproxy/envoy/blob/master/STYLE.md 

> The Google C++ style guide points out that constant vars should be named kConstantVar. In the Envoy codebase we use ConstantVar or CONSTANT_VAR. If you pick CONSTANT_VAR, please be certain the name is globally significant to avoid potential conflicts with #defines, which are not namespace-scoped, and may appear in externally controlled header files.